### PR TITLE
[ui] added a basic tooltip system

### PR DESCRIPTION
**Pull request by thomasdavis**

### CODE REVIEW

File: ui/app/src/components/shared/Tooltip.vue [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+export default defineComponent({
+  props: {
+    message: {
+      type: String,
+    },
+  },
+
+  data: function() {
+  	return {
+  		opened: false
```
Comment (iverc) [inline ui/app/src/components/shared/Tooltip.vue:13]: Dit auto-format it like that? could be as 1 line or have 2 spaces I think. @ryardley @59023g In general, I think we should have a default code style across the team so we don't restyle each other's code on every commit. It really makes commits look much bigger than they are and also makes it harder to review and easier to miss bugs.
What do you think about using prettier with default settings?

**Review by iverc — APPROVED** [review]

File: ui/app/src/components/shared/Tooltip.vue (hunk 1 above)
Comment (thomasdavis) [reply ui/app/src/components/shared/Tooltip.vue:13]: I am a super prettier fanboy, I would love if we can use it. 

I think they do a thorough job of explaining the benefits for the team -> https://prettier.io/docs/en/why-prettier.html

Comment (59023g) [reply ui/app/src/components/shared/Tooltip.vue:13]: I know @ryardley had a linter in mind. Prettier sounds good to me I'll ping the group in Slack. We will find consensus

File: ui/app/src/components/shared/Tooltip.vue [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    };
+  },
+  methods: {
+    close() {
+      console.log("close tool tips");
```
Comment (59023g) [inline ui/app/src/components/shared/Tooltip.vue:18]: would  remove this

**Review by 59023g — APPROVED** [review]
looks good!

**Pull request merged**